ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -128,9 +128,10 @@ class KtlintCommandLine {
 
     @Option(
         names = ["--color"],
-        description = ["Make output colorful"]
+        description = ["Make output colorful. To specify the color, use --COLOR=RED"],
```

Comment:
I think it's better to specify lowercase `--color` here, because picocli is case sensitive. You'll end up with `UnmatchedArgumentException` otherwise.